### PR TITLE
Sponge integration

### PR DESCRIPTION
**Pull request by s0rcerer88**
SpongeVanilla integration

### CODE REVIEW

**Comment by s0rcerer88** [comment]
@itzg PR done.

File: minecraft-server/start-configuration [hunk 1]
```diff
@@ -73,6 +73,10 @@ case "$TYPE" in
     exec /start-deployVanilla $@
   ;;
 
+  SPONGEVANILLA|spongevanilla)
```
Comment (itzg) [inline minecraft-server/start-configuration]: Since I haven't used Sponge personally, would it be confusing to shorten this to `SPONGE` or `sponge`?

File: minecraft-server/start-minecraftFinalSetup [hunk 2]
```diff
@@ -24,12 +24,17 @@ if [ -n "$ICON" -a ! -e server-icon.png ]; then
 fi
 
 # Make sure files exist and are valid JSON (for pre-1.12 to 1.12 upgrades)
-for j in *.json; do
-  if [[ $(python -c "print open('$j').read().strip()==''") = True ]]; then
-    echo "Fixing JSON $j"
-    echo '[]' > $j
-  fi
-done
+if [[ -z "ls *.json" ]]; then
```
Comment (itzg) [inline minecraft-server/start-minecraftFinalSetup]: Thanks!

File: minecraft-server/start-configuration (hunk 1 above)
Comment (s0rcerer88) [reply minecraft-server/start-configuration]: Yeah. SPONGEFORGE also exists and is the next task in my list.

File: minecraft-server/start-minecraftFinalSetup (hunk 2 above)
Comment (s0rcerer88) [reply minecraft-server/start-minecraftFinalSetup]: You're welcome!!!! Just a little fix.

**Comment by s0rcerer88** [comment]
Ok. I can add it to the readme. Thx!!!!!

**Comment by s0rcerer88** [comment]
Added a short description for SpongeVanilla on README.

Let me know if you think I've missed something important @itzg

**Comment by s0rcerer88** [comment]
I think you can merge @itzg . I will open a new PR for SpongeForge integration.
Thanks!!!

**Comment by itzg** [comment]
It is now built and available on Docker Hub. Thank you so much for all the contributions so far.
